pkg/api/annotations.go

```diff
@@ -30,17 +33,32 @@ func GetAnnotations(c *middleware.Context) Response {
 	result := make([]dtos.Annotation, 0)
 
 	for _, item := range items {
+		// Get user info for annotation event
+		userName := ""
+		if item.UserId != 0 {
+			userQuery := &models.GetUserByIdQuery{
```

Comment:
this looks really bad, as it will issue one query PER annotation :) Think we need to join based on userId